src/fs.rs

```diff
@@ -190,7 +192,6 @@ impl PrivateAccess for AccessFs {
 }
 
 // TODO: Make ACCESS_FILE a property of AccessFs.
-// TODO: Add tests for ACCESS_FILE.
```

Comment:
This TODO is a reminder to check that `ACCESS_FILE` can be used on a file *and* that is not the case for its complement. For instance, if `IoctlDev` wasn't part of `ACCESS_FILE`, such a test should detect it. That would not be a unit test but an integration test, see #57.